src/components/dialog-manager.js

```diff
@@ -58,28 +58,32 @@ export class DialogManager {
   }
 
   /**
+   * @param {boolean=} animated
+   * @param {boolean=} hidden
    * @return {!Promise<!Dialog>}
    */
-  openDialog() {
+  openDialog(animated = true, hidden = false) {
```

Comment:
I think it's worth touching base on this a bit: but wouldn't `animated` and `hidden` be mutually exclusive? Did you consider have a separate API, e.g. `openHiddenDialog()` and `openHiddenView()`? Any thoughts on that?

Reply:
talked offline. removed animated.